nova/tests/unit/virt/vmwareapi/test_driver_api.py

```diff
@@ -1512,18 +1512,25 @@ def test_poll_rebooting_instances(self, mock_reboot,
         self.conn.poll_rebooting_instances(60, instances)
         mock_reboot.assert_called_once_with(mock.ANY, mock.ANY, mock.ANY)
 
+    @mock.patch.object(vmops.VMwareVMOps, '_get_instance_property')
     @mock.patch.object(vmops.VMwareVMOps, '_get_instance_props')
     @mock.patch.object(vmops.VMwareVMOps, 'update_cached_instances')
     def test_reboot_not_poweredon(self, mock_update_cached_instances,
-                                  mock_get_instance_props):
-        mock_get_instance_props.return_value = {
-            "runtime.powerState": "poweredOff",
+                                  mock_get_instance_props,
+                                  mock_get_instance_property):
+        values = {
+            "runtime.powerState": "poweredOn",
```

Comment:
This changed from `poweredOff` to `poweredOn` while the test is called `"test_reboot_not_poweredon" so it should be `poweredOff`.

But in general this test seems to test suspending a VM, not rebooting and maybe it was just buggy before.

Reply:
I think, the name is simply wrong. See the [test right below](https://github.com/sapcc/nova/pull/417/files/3e138948bd34a96ed4fa6e9415886864275fef1f#diff-b0e6330c15bb8d6c2ca496b3e19a4ee130b0af1c334edbfc4b32d7d576eb3206R1528-R1529) after creating the VM: